### ISSUE TITLE
Related benefits number is incorrect

### ISSUE THREAD

**Issue by brdunfield** [veteransaffairscanada/vac-benefits-directory#1523]
"Can provide access to x benefits"

x does not match the amount of benefits being rendered for most cards

**Comment by smcmurtry** [comment]
x is the count of unique benefits, since some could be present in the family member list and the service member list. Also, this count is going away in the new card design: 

<img width="393" alt="screen shot 2018-12-04 at 9 27 01 am" src="https://user-images.githubusercontent.com/5498428/49445039-4724e700-f7a7-11e8-9552-5d8ee26193b4.png">

**Comment by brdunfield** [comment]
Will not be an issue after #1524 is merged
